test/unit/event/verification-method/hcs-did-revoke-verification-method-event.test.ts

```diff
@@ -0,0 +1,89 @@
+import { PrivateKey } from "@hashgraph/sdk";
+import { DidError, Hashing, HcsDidEventTargetName, HcsDidRevokeVerificationMethodEvent } from "../../../../dist";
+
+describe("HcsDidRevokeVerificationMethodEvent", () => {
+    const privateKey = PrivateKey.fromString(
+        "302e020100300506032b6570042204209044d8f201e4b0aa7ba8ed577b0334b8cb6e38aad6c596171b5b1246737f5079"
+    );
+    const identifier = `did:hedera:testnet:${Hashing.multibase.encode(privateKey.publicKey.toBytes())}_0.0.29613327`;
+    const event = new HcsDidRevokeVerificationMethodEvent(identifier + "#key-1");
```

Comment:
hardcoded account id

Reply:
It is a topic_id ; needs to be known for test cases to inspect outputs